soc/arm/silabs_exx32/common/pinctrl_soc.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2022 Silicon Labs
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * Silabs SoC specific helpers for pinctrl driver
+ */
+
+#ifndef ZEPHYR_SOC_ARM_SILABS_GECKO_COMMON_PINCTRL_SOC_H_
+#define ZEPHYR_SOC_ARM_SILABS_GECKO_COMMON_PINCTRL_SOC_H_
+
+#include <zephyr/devicetree.h>
+#include <zephyr/dt-bindings/pinctrl/gecko-pinctrl.h>
+#include <zephyr/types.h>
```

Comment:
`<stdint.h>`?

Reply:
Replaced include: `stdint.h` instead of `types.h`